templates/16-04/index.html

```diff
@@ -336,8 +330,8 @@ <h2>Watch the 20.04 migration webinar</h2>
     <div class="col-6">
       <h2>Ubuntu 16.04 LTS kernel lifecycle and enablement stack</h2>
       <p>The Ubuntu LTS enablement, or Hardware Enablement (HWE), stacks provide the newer kernel and X.Org support for existing Ubuntu LTS releases. These stacks can be installed manually, but are also installed with an Ubuntu LTS point release.</p>
-      <p>The 16.04.2 and newer point releases will ship with an updated kernel and X.Org stack by default for the desktop. Server installations will default to the General Availability (GA) kernel and provide the enablement kernel as optional.</p>
-      <p>The 16.04 HWE Stacks will follow a new Rolling Update Model. It is recommended to read the documentation before executing the following commands, as the HWE model has changed with Ubuntu 16.04 LTS.</p>
+      <p>The 16.04.2 and newer point releases ship with an updated kernel and X.Org stack by default for the desktop. Server installations default to the General Availability (GA) kernel and provide the enablement kernel as optional.</p>
+      <p>Ubuntu 16.04 ESM includes support for v4.4 and v4.15 based generic, lowlatency, hwe-16.04, aws and azure Linux kernel flavours.</p>
```

Comment:
I think all this should be capitilised or uppercase like in the rest of the text (AWS, HWE-16.04, Azure) but I wrote a comment in the [copy doc](https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#heading=h.7nyqndmsf52j).

Reply:
And I wonder if `lowlatency` should be stuck together?